tests/end_to_end/client.py

```diff
@@ -21,7 +21,7 @@ def __init__(self, serverUrl, flags=None):
         self.outFile = None
         self.errFile = None
         if flags is None:
-            self.flags = "-v"
+            self.flags = "-vv"
```

Comment:
For some reason, the logs emitting POST messages are only there at a higher log level now and tests were failing because of this. I don't really know why this has changed, but I hope we'll be changing this code around soon enough so I didn't worry too much about it.